app/classes/Update.php

```diff
@@ -584,22 +589,40 @@ private function schema22()
     }
 
     /**
-     * Add todolist table per experiment
+     * Change column type of body in 'items_revisions' and 'experiments_revisions' to 'mediumtext'
+     * See elabftw/elabftw#429
      *
+     * @throws Exception
      */
     private function schema23()
     {
+        $sql = "ALTER TABLE experiments_revisions MODIFY body MEDIUMTEXT";
+        $sql2 = "ALTER TABLE items_revisions MODIFY body MEDIUMTEXT";
+
+        if (!$this->pdo->q($sql)) {
+            throw new Exception('Cannot change type of column "body" in table "experiments_revisions"!');
+        }
+        if (!$this->pdo->q($sql2)) {
+            throw new Exception('Cannot change type of column "body" in table "items_revisions"!');
+        }
+    }
+  
+    /**
+     * Add todolist table per experiment
+     *
+     */
+    private function schema24()
+    {
         $sql = "CREATE TABLE IF NOT EXISTS `experiments_steps` (
             `id` int(10) UNSIGNED NOT NULL AUTO_INCREMENT,
             `experiment_id` int(10) UNSIGNED NOT NULL,
             `body` text NOT NULL,
             `creation_time` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
             `ordering` int(10) UNSIGNED DEFAULT NULL,
             `userid` int(10) UNSIGNED NOT NULL,
-            PRIMARY KEY (`id`)
-            ) ENGINE=InnoDB DEFAULT CHARSET=utf8;";
```

Comment:
@NicolasCARPi After working on issue #433 I do not think it is a good idea to change the default behavior of the tables, so I removed the ENGINE and DEFAULT CHARSET parameters.